### PR TITLE
More work done on display results

### PR DESCRIPTION
**Pull request by LogicX1**
rel #19 

Changed api url with cors-anywhere  prefix as proxy .
Added and calculated the time for the api request to finish.
Started working on displaying one course , replacing <link> method with a state method,
because we cannot send props (a big course object) in the link.

### CODE REVIEW

File: client/src/components/Search/Search.js [hunk 1]
```diff
@@ -9,27 +9,36 @@ const Search = props => {
   const [error, setError] = useState(null);
   const [loading, setLoading] = useState(true);
   const [searchResult, setSearchResult] = useState(null);
-  const { term } = props.match.params;
+  const [term, setTerm] = useState(props.match.params.term);
+  const reqTime = React.useRef(0);
   const apiBase = 'https://www.udemy.com/api-2.0';
+  console.log(term);
```
Comment (Karyum) [inline client/src/components/Search/Search.js]: no console logs for frontend.

also please add new lines between lines of code.

File: client/src/pages/CoursePage.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
 import React from 'react';
+
 import SelectedCourse from '../components/SelectedCourse/SelectedCourse';
 
 
-function CoursePage() {
+
+function CoursePage({ course }) {
+  console.log(this.course);
```
Comment (Karyum) [inline client/src/pages/CoursePage.js]: again remove this

File: client/src/utils/fetchGet.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const fetchGet = url => {
+  const myHeaders = new Headers({
+    Accept: 'application/json, text/plain, */*',
+    Authorization: `Basic ${process.env.REACT_APP_AUTH}`,
+    'Content-Type': 'application/json;charset=utf-8',
+  });
+  const config = {
+    // mode: 'no-cors',
+    // credentials: 'include',
+    // origin: 'https://github.com/FACN8/Learnow',
```
Comment (Karyum) [inline client/src/utils/fetchGet.js]: do you need these anymore ?

File: client/src/pages/CoursePage.js [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
-      url =  "https://www.udemy.com/course/algebra-i-beginning-algebra/"
-      image = "https://i.udemycdn.com/course/240x135/34267_f952_15.jpg"
+
+    <SelectedCourse id = {1} title = {course.title}
+      headline = {course.headline}
+      url = {course.url}
+      image = {course.image_240x135}
+
       num_lectures = {14} num_subscribers = {566}
     />
```
Comment (Karyum) [inline client/src/pages/CoursePage.js]: so this component `CoursePage` only renders `SelectedCourse` will it render anything else in the future?

Comment (LogicX1) [reply client/src/pages/CoursePage.js]: of course, it will render more course data, and groups

**Pull request merged**